app/src/main/java/com/mealsmadeeasy/model/Recipe.kt

```diff
@@ -0,0 +1,11 @@
+package com.mealsmadeeasy.model
+
+data class Recipe(
+        val prepTime: Int, // Minutes
```

Comment:
There's another file called `Units.kt`. Let's go ahead and add a type alias here for `Minutes = Int`. That way, IntelliJ gives us hints about the unit and it looks cool.